codegen/templates/endpoint.tmpl

```diff
@@ -50,21 +52,20 @@ func Handle{{title .Name}}Request(
 	}
 	{{end}}
 
-	headers := map[string]string{}
```

Comment:
I think moving to map[string]string is going to hurt things in the long run because we lose canonicalization of headers, so the request response can become unmatched based on capitalization